diff --git a/py/misc.h b/py/misc.h
index a9c02a4dd..924688871 100644
--- a/py/misc.h
+++ b/py/misc.h
@@ -81,7 +81,7 @@ typedef unsigned int uint;
 // do, others only with -Werror=vla, msvc always does.
 // The (void) is needed to avoid "left operand of comma operator has no effect [-Werror=unused-value]"
 // when using this macro on the left-hand side of a comma.
-#if defined(_MSC_VER) || defined(__cplusplus)
+#if defined(_MSC_VER) || defined(__cplusplus) || defined(__CALYPSI__)
 #define MP_STATIC_ASSERT_NONCONSTEXPR(cond) ((void)1)
 #else
 #if __clang__
